hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/JobController.java

```diff
@@ -68,8 +70,19 @@ public Jobs createJob(Users user, Project project, JobConfiguration config) thro
       throw new JobException(RESTCodes.JobErrorCode.JOB_NAME_EXISTS, Level.FINE,
         "Job with name:" + config.getAppName() + " for " + "project:" + project.getName() + " already exists");
     }
-    
-    Jobs created = jobFacade.create(user, project, config);
+
+    Jobs created;
+    try {
+      created = jobFacade.create(user, project, config);
+    } catch(RuntimeException re) {
+      if(re.getCause() instanceof JAXBException) {
+        throw new JobException(RESTCodes.JobErrorCode.JOB_CONFIGURATION_CONVERT_TO_JSON_ERROR, Level.SEVERE,
```

Comment:
@robzor92 when we catch exception and then throw RESTExceptions, must also set the dev message and the throwable in the RESTException constructor. In this case it should be 
`throw new JobException(RESTCodes.JobErrorCode.JOB_CONFIGURATION_CONVERT_TO_JSON_ERROR, Level.SEVERE, "Unable to create json from JobConfiguration", re.getMessage(), re);`